capstone-app/src/app/modules/history/wiki-media/wiki-media.component.spec.ts

```diff
@@ -19,10 +19,14 @@ describe('WikiMediaComponent', () => {
         parse: {
           title: 'Stillwater, Oklahoma',
           text: {
-            '*' : "</h2><p>The north-central region of Oklahoma became part of the United States with the <a target=\"_blank\" href=\"https://en.wikipedia.org/wiki/Louisiana_Purchase\" title=\"Louisiana Purchase\">Louisiana Purchase</a> in 1803. In 1832, author and traveler <a target=\"_blank\" href=\"https://en.wikipedia.org/wiki/Washington_Irving\" title=\"Washington Irving\">Washington Irving</a> provided the first recorded description of the area around Stillwater in his book <i>A Tour on the Prairies</i>. He wrote of “a glorious prairie spreading out beneath the golden beams of an autumnal sun. The deep and frequent traces of buffalo, showed it to be a one of their favorite grazing grounds.”<sup id=\"cite_ref-9\" class=\"reference\"><a href=\"#cite_note-9\">&#91;9&#93;</a></sup></p>"
-          }
+            '*' : 'The north-central region of Oklahoma became part of the United States with the Louisiana '
+            + 'Purchase in 1803. In 1832, author and traveler Washington Irving provided the first recorded '
+            + 'description of the area around Stillwater in his book A Tour on the Prairies. He wrote of “a '
+            + 'glorious prairie spreading out beneath the golden beams of an autumnal sun. The deep and frequent '
+            + 'traces of buffalo, showed it to be a one of their favorite grazing grounds.”9'
         }
       }
+    }
 
     const wikiService = jasmine.createSpyObj('WikiResultsService', ['search']);
     getWikiResponseSpy = wikiService.search.and.returnValue(of(MockWikiResponse));
```

Comment:
On l. 32 and 49 it looks like we call and.returnValue in both places on the same method for the service.
The one that we want is the one on l. 32, not l.49, because the service call does not remove the tags, so that one is closer to how the service actually behaves.
The text formatted in the testWikiResult object has the tags removed, which is how the string should display in the body  after being parsed by the component's fixString method (you use it in l.59 correctly as the toEqual argument).